### ISSUE TITLE
Expert Guide}FAQ}}Can I talk to people on Robinhood?

### ISSUE THREAD

**Issue by makalombrandam** [yoyosaredore/service#76]
Can I talk to people on Robinhood?

(𝓒𝓪𝓵𝓵 24/7 𝓤𝓢) 𝓨𝓮𝓼, ＋１1844 724 5399))> －[*+<1844 724 5399))> ] 𝔂𝓸𝓾 𝓬𝓪𝓷 𝓽𝓪𝓵𝓴/𝓼𝓹𝓮𝓪𝓴/𝓬𝓱𝓪𝓽 𝓭𝓲𝓻𝓮𝓬𝓽𝓵𝔂 𝓽𝓸 𝓹𝓮𝓸𝓹𝓵𝓮 𝓸𝓷 𝓡𝓸𝓫𝓲𝓷𝓱𝓸𝓸𝓭 𝓼𝓾𝓹𝓹𝓸𝓻𝓽 𝓷𝓾𝓶𝓫𝓮𝓻 𝓫𝔂 𝓬𝓸𝓷𝓽𝓪𝓬𝓽𝓲𝓷𝓰 𝓽𝓱𝓮𝓲𝓻 𝓡𝓸𝓫𝓲𝓷𝓱𝓸𝓸𝓭 𝓒𝓾𝓼𝓽𝓸𝓶𝓮𝓻 24*7 𝓗𝓮𝓵𝓹𝓵𝓲𝓷𝓮 𝓷𝓾𝓶𝓫𝓮𝓻 ＋１－[*+<1844 724 5399))> ] 𝓕𝓸𝓻 𝓺𝓾𝓲𝓬𝓴 𝓪𝓼𝓼𝓲𝓼𝓽𝓪𝓷𝓬𝓮, 𝓽𝓱𝓮𝓲𝓻 24*7 𝓗𝓸𝓽𝓵𝓲𝓷𝓮 𝓷𝓾𝓶𝓫𝓮𝓻 𝓲𝓼 𝓪𝓿𝓪𝓲𝓵𝓪𝓫𝓵𝓮 24/7.
No, you cannot directly talk 🐾 🎀 *+<1844 724 5399))> }} 🎀 🐾 to other users on Robinhood. Robinhood is a trading platform focused on buying and selling stocks, and it does not include social features like chatting with other users
Phone Support: 📞🐾 *+<1844 724 5399))> }} 🎀 🐾Robinhood offers phone support, where users can request a callback instead of waiting on hold. Response time can range from a few minutes to a few hours during business hours*+<1844 724 5399))> . For pressing issues, response times are generally quicker.
